src/olympia/users/management/commands/createsuperuser.py

```diff
@@ -81,6 +89,11 @@ def handle(self, *args, **options):
                 for field_name in self.required_fields
             }
 
+        if options.get('fxa_id', False):
```

Comment:
`None` is a more reasonable default value since it represents "no value" rather than just being `False` (same for the `parser.add_argument` call

Reply:
So I should change the ```--hostname``` option as well huh.